src/sysl/importers/import_swagger.py

```diff
@@ -147,137 +145,208 @@ def translate_path_template_params(self, path):
         return re.sub(r'({[^/]*?})', self.javaParam, path)
 
     def translate(self, swag, appname, package, w):
+        hasInfo = False
 
         if 'info' in swag:
-            def info_attrs(info, prefix=''):
-                for (name, value) in sorted(info.iteritems()):
-                    if isinstance(value, dict):
-                        info_attrs(value, prefix + name + '.')
-                    else:
-                        w('@{}{} = {}', prefix, name, json.dumps(value))
-
             title = swag['info'].pop('title', '')
-            info_attrs(swag['info'])
+            hasInfo = True
```

Comment:
```python
hasInfo = 'info' in swag
title = swag['info'].pop('title', '') if hasInfo else ''
```
While we're at it, why do we pop 'title' from the dict?

Reply:
If it is not removed, title will be added as an attribute as well which I think is not required.

```
 "Simple" [package=""]:
    @title = "Simple"
    @description =:
        | No description.
```